src/controllers/userController.js

```diff
@@ -276,3 +279,9 @@ export async function deleteUser(req, res) {
     });
   }
 }
+export async function logoutUser(req, res) {
+  return res.status(200).json({
+    status: 200,
+    message: 'User logged out successfully',
+  });
```

Comment:
I think this is not enough to logout the user, would you try to improve this process?